src/Eto/Eto.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DefineConstants>TRACE;DEBUG;PCL</DefineConstants>
+    <OutputPath>..\..\artifacts\core\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DefineConstants>TRACE;PCL</DefineConstants>
+    <OutputPath>..\..\artifacts\core\Release\</OutputPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Shared\EnumerableExtensions.cs" />
+    <Compile Include="..\Shared\GlobalAssemblyInfo.cs" />
```

Comment:
You can ether create a Eto.props file with stuff that's in AssemblyInfo.cs and import them, or you can add AssemblyInfo properties to csproj itself example:
```xml
    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
        <DebugType>Full</DebugType>
        <Title>MyApp</Title>
        <Authors>MyCompany</Authors>
        <Description>My Description</Description>
        <Copyright>My Copyright</Copyright>
        <Version>1.0.0.0</Version>
    </PropertyGroup>
```
These are also added into nuget package.
For globals we do:
MyProduct.targets:
```xml
<Project>
    <PropertyGroup>
        <Authors>My Company</Authors>
        <Owners>$(Authors)</Owners>
        <Company>$(Authors)</Company>
        <CopyrightYear>2018</CopyrightYear>
        <Copyright>(c) 2010-$(CopyrightYear) by Curtis Wensley, 2012-2014 by Vivek Jhaveri and contributors</Copyright>
        <Product>My Product</Product>
        <PackageIconUrl>url to nuget icon</PackageIconUrl>
        <RepositoryUrl>Path to repository</RepositoryUrl>
        <RepositoryType>Git</RepositoryType>
        <GenerateDocumentationFile>true</GenerateDocumentationFile>
        <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
    </PropertyGroup>
</Project>
```
MyProject.csproj:
`<Import Project="../path/to/MyProduct.targets" />`

Reply:
Yeah I know about this, but since I need the GlobalAssemblyInfo for all the other non-sdk style projects I won't be able to do this afaik.  Also, without multi-targeting support in VS for Mac I won't be able to use it for the xaml/json assemblies either so I will keep the nuspec files for now.  I may transition to this later when the tooling catches up..

Reply:
Yea, multi targeting support is not available on Linux either, tho this is not a bug in VS for Mac/MD, its a bug with msbuild. There are 2 msbuilds that get installed and neither of them are able to see the framework the other one supports:
 - `msbuild` - only sees standard .NET Framewrok
 - `dotnet msbuild` - only sees .NET Core frameworks

On a side note, they are also linked against only the framework they support so you actually have to build 2 C# task files if you want to do custom tasks.